default.yml

```diff
@@ -543,18 +543,7 @@ Style/EmptyMethod:
     - compact
     - expanded
 
-# Checks whether the source file has a utf-8 encoding comment or not
-# AutoCorrectEncodingComment must match the regex
-# /#.*coding\s?[:=]\s?(?:UTF|utf)-8/
-Style/Encoding:
```

Comment:
We don't want this anymore? Or is it no longer supported?

Reply:
No longer supported